doc/source/index.rst

```diff
@@ -73,6 +73,7 @@ Wherever possible, do not take these conversations to private channels, includin
    examples/quickstart
    faq
    examples/README
+   releases
```

Comment:
Can probably move this around to a more prominent location - just couldn't really think of one right now